websites/E/Emby/presence.ts

```diff
@@ -732,16 +740,19 @@ async function handleWebClient(): Promise<void> {
  * setDefaultsToPresence - set defaul values to the presenceData object
  */
 async function setDefaultsToPresence(): Promise<void> {
+	presenceData.largeImageKey = PRESENCE_ART_ASSETS.logo;
+
 	if (presenceData.smallImageKey) delete presenceData.smallImageKey;
 
 	if (presenceData.smallImageText) delete presenceData.smallImageText;
 
 	if (presenceData.startTimestamp) delete presenceData.startTimestamp;
 
-	if (presenceData.endTimestamp) delete presenceData.endTimestamp;
+	if (presenceData.endTimestamp && isNaN(presenceData.endTimestamp))
```

Comment:
First check is kinda redundant, you can just check for isNaN

Reply:
Removed first check of the condition